pages/c/_cluster/settings/brand.vue

```diff
@@ -112,7 +116,33 @@ export default {
 
       uiCommunitySetting: {},
 
-      errors: []
+      errors: [],
+
+      uiBannerFontSizeOptions: [{
+        label: '10px',
+        value: '10px'
+      },
+      {
+        label: '12px',
+        value: '12px'
+      },
+      {
+        label: '14px',
+        value: '14px'
+      },
+      {
+        label: '16px',
+        value: '16px'
+      },
+      {
+        label: '18px',
+        value: '18px'
+      },
+      {
+        label: '20px',
+        value: '20px'
+      }
+      ]
```

Comment:
A small suggestion would be to change this to an array of strings without the `label` and `value` keys, then just remove the `option-key` and `option-label` from the two `LabeledSelect` components. Otherwise LGTM

Reply:
Updated :).